lib/blockscore/base.rb

```diff
@@ -4,19 +4,37 @@ module BlockScore
   class Base
     extend Connection
 
-    attr_reader :attributes
-
-    def initialize(options = {})
+    def initialize(options = {}, &block)
+      @loaded = !(block)
```

Comment:
I'm not sure I understand what is going on here.  I looked at the inheriting classes and I can't seem to find a block being provided